src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -218,7 +218,7 @@ private async Task InitializeAsync()
             _solutionEvents.ProjectRemoved += OnEnvDTEProjectRemoved;
             _solutionEvents.ProjectRenamed += OnEnvDTEProjectRenamed;
 
-            var vSStd97CmdIDGUID = VSConstants.GUID_VSStandardCommandSet97.ToString("B", CultureInfo.CurrentCulture);
+            var vSStd97CmdIDGUID = VSConstants.GUID_VSStandardCommandSet97.ToString("B", CultureInfo.InvariantCulture);
```

Comment:
I left a comment in a previous PR after it had been merged (mb), I think we should ignore the warnings raised for Guid ToString as this has the potential to break eeventually.

Reply:
I think you mean to change it to `InvariantCulture` but I can add the suppressions for this. Or I can pass null to the method call. https://docs.microsoft.com/en-us/dotnet/api/system.guid.tostring?view=net-6.0#system-guid-tostring(system-string-system-iformatprovider)

Reply:
null's fine if we mean to avoid the warning